doc/src/TEAL_user_manual.tex

```diff
@@ -0,0 +1,196 @@
+\definecolor{DPMcolor}{HTML}{E28B8D} % Dan
+\definecolor{JCcolor}{HTML}{82A8D9}  % Josh (adjusted for clarity)
+\definecolor{AAcolor}{HTML}{8D7F44}  % Andrea
+\definecolor{CRcolor}{HTML}{AC39CE}  % Cristian
+\definecolor{RKcolor}{HTML}{3ECC8D}  % Bob (adjusted for clarity)
+\definecolor{DMcolor}{HTML}{276605}  % Diego (adjusted for clarity)
+\definecolor{PTcolor}{HTML}{990000}  % Paul
+\def\DRAFT{} % Uncomment this if you want to see the notes people have been adding
+% Comment command for developers (Should only be used under active development)
+\ifdefined\DRAFT
+  \newcommand{\nameLabeler}[3]{\textcolor{#2}{[[#1: #3]]}}
+\else
+  \newcommand{\nameLabeler}[3]{}
+\fi
+\newcommand{\alfoa}[1] {\nameLabeler{Andrea}{AAcolor}{#1}}
+\newcommand{\cristr}[1] {\nameLabeler{Cristian}{CRcolor}{#1}}
+\newcommand{\talbpaul}[1] {\nameLabeler{Paul}{PTcolor}{#1}}
+% Commands for making the LaTeX a bit more uniform and cleaner
+\newcommand{\TODO}[1]    {\textcolor{red}{\textit{(#1)}}}
+\newcommand{\xmlAttrRequired}[1] {\textcolor{red}{\textbf{\texttt{#1}}}}
+\newcommand{\xmlAttr}[1] {\textcolor{cyan}{\textbf{\texttt{#1}}}}
+\newcommand{\xmlNodeRequired}[1] {\textcolor{deepblue}{\textbf{\texttt{<#1>}}}}
+\newcommand{\xmlNode}[1] {\textcolor{darkblue}{\textbf{\texttt{<#1>}}}}
+\newcommand{\xmlString}[1] {\textcolor{black}{\textbf{\texttt{'#1'}}}}
+\newcommand{\xmlDesc}[1] {\textbf{\textit{#1}}} % Maybe a misnomer, but I am
+                                                % using this to detail the data
+                                                % type and necessity of an XML
+                                                % node or attribute,
+                                                % xmlDesc = XML description
+\newcommand{\default}[1]{~\\*\textit{Default: #1}}
+\newcommand{\nb} {\textcolor{deepgreen}{\textbf{~Note:}}~}
+\usepackage{bm}
+\newcommand{\tensor}[1]{{\bm{#1}}}
+\renewcommand{\vec}{\bm}
+\newcommand{\unit}[1]{\hat{\bm{#1}}}
+\newcommand{\scalarunit}[1]{\hat{#1}}
+\usepackage{booktabs}
+\usepackage{stmaryrd}
+\usepackage{hyperref}
+\hypersetup{
+    colorlinks,
+    citecolor=black,
+    filecolor=black,
+    linkcolor=black,
+    urlcolor=black
+}
+\newcommand{\wiki}{\href{https://github.com/idaholab/raven/wiki}{RAVEN wiki}}
+\usepackage{cite}
+\raggedbottom
+\setcounter{secnumdepth}{5} % show 5 levels of subsection
+\setcounter{tocdepth}{5} % include 5 levels of subsection in table of contents
+\title{Tool for Economic AnaLysis (Teal) User Manual \\
+    \large Economics plugin for RAVEN \\}
+%\author{
+%\textbf{\textit{Project Manager:}}
+% \\Cristian Rabiti\\
+% \textbf{\textit{Principal Investigator and Technical Leader:}}
+%\\Paul W. Talbot\\
+%\textbf{\textit{Main Developers:}}
+%\\Paul W. Talbot\\
+%Abhinav Gairola\\
+%\textbf{\textit{Additional  Developers:}}
+%\\Jia Zhou
+%}
+
+\author{ \\Aaron S. Epiney, Paul Talbot, Congjian Wang, Andrea Alfonsi, Elizabeth Worsham\\}
+
+\date{}
+\SANDnum{INL/EXT-00-00000, GDE-000}
```

Comment:
this should probably be updated

Reply:
What is the report number/version? It wasn't listed on the previous manual

Reply:
I don't know. It should be in LRS, or maybe Andrea has it; I was not involved, I don't think, in that.

Reply:
It looks like this value should be filled in a future PR.